### PR TITLE
Last min Mp4 fixes

### PR DESCRIPTION
**Pull request by johnnyd-eth**
## What does this PR do and why?

- reduce file upload limit
- improve loading state
- allow .mov files
- Fix wide mp4 preview on upload 

https://user-images.githubusercontent.com/96150256/220289878-1d5644eb-c7fc-43e3-aeba-2db209a87770.mp4

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/RichNote/hooks/loadAllMediaLinks.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ const supportedContentTypes = [
   'image/png',
   'image/svg',
   MP4_FILE_TYPE,
+  MOV_FILE_TYPE,
```
Comment (tomquirk) [inline src/components/RichNote/hooks/loadAllMediaLinks.ts:11]: **suggestion** could we have a single constant for VIDEO_FILE_TYPES and use that around the place? bit of duplication here

**Review by tomquirk — APPROVED** [review]

File: src/components/Create/components/RewardsList/AddEditRewardModal.tsx [hunk 2]
```diff
@@ -44,7 +44,8 @@ interface AddEditRewardModalFormProps {
   externalUrl?: string | undefined
 }
 
-const NFT_FILE_UPLOAD_EXTRA = t`NFT will be cropped to a 1:1 square in thumbnail previews on the Juicebox app.`
+const NFT_FILE_UPLOAD_EXTRA = t`Images will be cropped to a 1:1 square in thumbnail previews on the Juicebox app.`
+const MAX_NFT_FILE_SIZE_MB = 100
```
Comment (tomquirk) [inline src/components/Create/components/RewardsList/AddEditRewardModal.tsx:48]: @johnnyd-eth shit this actually needs to be `100 * 1024 * 1024` apparently. pls confirm

File: src/components/RichNote/hooks/loadAllMediaLinks.ts (hunk 1 above)
Comment (johnnyd-eth) [reply src/components/RichNote/hooks/loadAllMediaLinks.ts:11]: Ye yup gr8 minds think alike, did that earlier

File: src/components/Create/components/RewardsList/AddEditRewardModal.tsx (hunk 2 above)
Comment (johnnyd-eth) [reply src/components/Create/components/RewardsList/AddEditRewardModal.tsx:48]: Nup, as is is good. We reduce the fileSize in later checks:

Screen Shot 2023-02-21 at 9 01 49 pm

Have changed the `sizeLimit` prop to `sizeLimitMB` as suggested in Discord.

**Pull request merged**